downloads.md

```diff
@@ -32,6 +32,27 @@ the “master” branch on GitHub.
 We try to keep things on the master branch fairly stable and tested,
 but there are bound to be bugs.
 
+MyPaint is now also availble as Flatpak and should be installable on
+all major Linux distributions that support it like Fedora, Debian,
+Ubuntu, elementaryOS, Arch, openSuSE, and many more.
+
+[Click to install](https://raw.githubusercontent.com/mypaint/mypaint/master/flatpak/mypaint-stable.flatpakref)
+
+Click to install is not yet availble in all distributions.  If you are
+lucky it will open your Software application.  Otherwise you can use
+the command line:
+
+```
+flatpak install --from https://raw.githubusercontent.com/mypaint/mypaint/master/flatpak/mypaint-stable.flatpakref
+```
+
+After installing the Flatpak, the applications will show up in your
+system, but you can also launch it from the commandline:
+
```

Comment:
@odysseywestra under which desktop? In GNOME Shell after installing a flatpak the desktop displays the launcher without needing to log out. But yes if there is at least one desktop not displaying the launcher automatically, is worth mentioning.